llama_stack/providers/tests/memory/test_memory.py

```diff
@@ -142,12 +142,14 @@ async def test_query_documents(memory_settings, sample_documents):
     assert_valid_response(response4)
     assert len(response4.chunks) <= 2
 
+    # Score threshold is not implemented in vector memory banks
```

Comment:
why was this commented out? did this fail?

Reply:
oh that's a bug could we fix that too please

Reply:
Hey @ashwinb.
I've updated the base implementation to propagate `score_threshold` and Qdrant uses it. This test is now uncommented and passes.